src/index.js

```diff
@@ -1,10 +1,16 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import { BrowserRouter } from "react-router-dom";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <BrowserRouter>
```

Comment:
I'm not sure it's good practice to have 2 lots of BrowserRouter in your application, it might cause some random bugs. I think it would be better to keep this one and remove the Router that you have inside of app.js? It would also solve Dex's query he raised